BraintreeCore/build.gradle

```diff
@@ -45,26 +45,26 @@ android {
 def playServicesWallet = "com.google.android.gms:play-services-wallet:${rootProject.playServicesWalletVersion}"
 
 dependencies {
-    implementation 'androidx.appcompat:appcompat:1.2.0'
-    implementation 'androidx.work:work-runtime:2.4.0'
+    implementation 'androidx.appcompat:appcompat:1.3.1'
+    implementation 'androidx.work:work-runtime:2.7.0-alpha05'
```

Comment:
Using a 2.7.0-alpha version of this library is required to fix an [Android 12 PendingIntent bug](https://developer.android.com/jetpack/androidx/releases/work#2.7.0-alpha02)